modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/asset/model/DDMFormAssetRenderer.java

```diff
@@ -152,6 +157,38 @@ public String getTitle(Locale locale) {
 			false);
 	}
 
+	@Override
+	public PortletURL getURLEdit(
+			LiferayPortletRequest liferayPortletRequest,
+			LiferayPortletResponse liferayPortletResponse)
+		throws Exception {
+
+		String portletNamespace =
+			DDMPortletKeys.DYNAMIC_DATA_MAPPING_FORM_ADMIN;
+
+		HttpServletRequest httpServletRequest =
+			liferayPortletRequest.getHttpServletRequest();
```

Comment:
@marcosapmf, as used rule. `httpServletRequest` is used first then `portletNamespace`.

Reply:
it's necessary to avoid a NPE [here](https://github.com/liferay/liferay-portal/blob/2152cecbba72d32572fa776486d7ea59bb0ce585/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormTemplateContextFactoryImpl.java#L126)